package-scripts.js

```diff
@@ -38,8 +46,22 @@ const truffleCommand = `node ${joinPath(pathArcJsRoot, truffleIsInternal ? "node
 module.exports = {
   scripts: {
     lint: {
-      default: "eslint .",
-      fix: 'nps "lint --fix ."'
+      default: series(
+        "nps lint.ts",
+        "nps lint.js"
+      ),
+      ts: {
+        default: "tslint --project .",
+        andFix: "nps \"lint.ts --fix\""
+      },
+      js: {
+        default: "eslint ./test",
+        andFix: "nps \"lint.js --fix\""
+      },
+      andFix: series(
```

Comment:
does fix runs by travis ?
I am not sure it is good idea for autofix .

Reply:
No, travis doesn't do the auto fix.